rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/util/ValidationUtil.java

```diff
@@ -83,18 +83,15 @@
 
 public class ValidationUtil {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)